src/transformers/models/phi/convert_phi_weights_to_hf.py

```diff
@@ -104,27 +102,48 @@ def _download(url: str, root: str):
     )
 
 
-def convert_phi_weights(checkpoint_path, pytorch_dump_folder_path, use_cuda, save_weights_directly):
+def convert_phi_weights(
+    model_name, checkpoint_path, pytorch_dump_folder_path, use_cuda, save_weights_directly, _MODELS
+):
+    _MODELS = _MODELS if model_name not in _MODELS.keys() else {model_name: _MODELS.get(model_name)}
```

Comment:
```suggestion
    model_name, checkpoint_path, pytorch_dump_folder_path, use_cuda, save_weights_directly):
    pretrained_models = _MODELS if model_name not in _MODELS.keys() else {model_name: _MODELS.get(model_name)}
```
and just use `pretrained_models`